src/components/CustomizableCalendarDay.jsx

```diff
@@ -207,6 +202,15 @@ class CustomizableCalendarDay extends React.Component {
 
     const isOutsideRange = modifiers.has('blocked-out-of-range');
 
+    const formattedDate = { date: day.format(ariaLabelFormat) };
+
+    let ariaLabel = getPhrase(chooseAvailableDate, formattedDate);
```

Comment:
Can you think of any clean way to share this logic with the identical code in CalendarDay?

Reply:
There's a lot more than just this duplicated between CalendarDay and CustomizableCalendarDay now.  I could put together a utility function somewhere though and call it from each to get the ariaLabel value.